GenderPayGap.WebUI/Views/Admin/ViewOrganisation.cshtml

```diff
@@ -179,10 +179,19 @@
         <tr class="govuk-table__row">
             <th scope="row" class="govuk-table__header">Scope</th>
             <td class="govuk-table__cell">
-                <b>
-                    @(Model.LatestScope.SnapshotDate.ToString("yyyy")):
-                </b>
-                @(Model.LatestScope.ScopeStatus)
+                @{
+                    var scopes = Model.OrganisationScopes
+                        .Where(s => s.Status == ScopeRowStatuses.Active)
+                        .OrderByDescending(s => s.SnapshotDate);
+                }
+                @foreach (var scope in scopes)
+                {
+                    <b>
+                        @($"{scope.SnapshotDate.Year}-{(scope.SnapshotDate.Year + 1) % 100}"):

```

Comment:
This is neat, but took me a second to figure out what is going on. Could we add a comment, or pull out a method for this formatting with a nice, descriptive method name?

Reply:
I looks like we would re-use it a few times, so maybe a method over a comment makes sense?